Templates/Per Tenant Extension/.AL-Go/Current.settings.json

```diff
@@ -0,0 +1,6 @@
+{
+    "artifact":  "////latest",
+    "cacheImageName":  "",
+    "appBuild":  2147483647,
```

Comment:
should it be checked in ?

Reply:
Yes, when running these current, nextminor and nextmajor, the default settings are in these json files